src/functional/geometry/convex_hull.zig

```diff
@@ -204,3 +345,77 @@ test "incremental convex hull random elements" {
         try testing.expectEqual(turn, Turn.left);
     }
 }
+
+test "Compute MABR LinearFunction for known ConvexHull one" {
```

Comment:
Maybe add a space before Hull for consistency.

Reply:
fixed